### PR TITLE
Bookworm Support

### PR DESCRIPTION
**Pull request by Tagadda**
## Solution

- Update dependencies for bookworm
- Remove buster dependencies as they are not needed anymore
- `jellyfin-ffmpeg6`

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Tagadda** [comment]
!bookwormtestme

**Comment by Tagadda** [comment]
!bookwormtestme

**Comment by Tagadda** [comment]
!testme

**Comment by Tagadda** [comment]
!bookwormtestme

File: scripts/_common.sh [hunk 1]
```diff
@@ -87,8 +87,8 @@ ffmpeg_deps_bookworm=(
 )
 
 case "$debian" in
-	bullseye) ffmpeg_deps=$ffmpeg_deps_bullseye ;;
- 	bookworm) ffmpeg_deps=$ffmpeg_deps_bookworm ;;
+	bullseye) ffmpeg_deps="${ffmpeg_deps_bullseye[*]}" ;;
+ 	bookworm) ffmpeg_deps="${ffmpeg_deps_bookworm[*]}" ;;
```
Comment (Salamandar) [inline scripts/_common.sh:92]: Nuuuuuu

`ffmpeg_deps=("${ffmpeg_deps_bookworm[@]}")`

allows you to keep it as a list

(yeah okay you don't care about this as it's transformed as a string just after)

**Review by Salamandar — APPROVED** [review]

File: scripts/_common.sh (hunk 1 above)
Comment (Tagadda) [reply scripts/_common.sh:92]: baaaaaaaaaaaaaaaaaaaaaaaaaaaaash /o\

**Comment by Tagadda** [comment]
Still failing to start on bookworm at this point...
```
95329 WARNING The service jellyfin didn't fully executed the action stop before the timeout.
95339 WARNING Please find here an extract of the end of the log of the service jellyfin:
95361 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] SsdpCommunicationsServer Disposing 3 sendSockets
95361 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] SsdpCommunicationsServer disposing sendSocket from 0.0.0.0
95361 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] SsdpCommunicationsServer disposing sendSocket from 10.130.112.228
95361 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] SsdpCommunicationsServer disposing sendSocket from 127.0.0.1
95361 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing LibraryMonitorStartup
95361 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing MusicBrainzAlbumProvider
95362 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing LibraryChangedNotifier
95362 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing UdpServerEntryPoint
95362 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing UserDataChangeNotifier
95362 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing SchedulesDirect
95362 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing EmbyTV
95362 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing DeviceAccessEntryPoint
95362 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing ExternalPortForwarding
95362 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Stopping NAT discovery
95362 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing EntryPoint
95363 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing RecordingNotifier
95363 WARNING Sep 26 16:24:43 jellyfin[4024]: [16:24:43] [INF] Disposing NotificationEntryPoint
95363 WARNING Sep 26 16:24:43 systemd[1]: jellyfin.service: Deactivated successfully.
95363 WARNING Sep 26 16:24:43 systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
95363 WARNING Sep 26 16:24:43 systemd[1]: jellyfin.service: Consumed 7.032s CPU time.
```

**Pull request closed without merging**